README.md

```diff
@@ -84,6 +84,31 @@ rules: {
 6. Use the warnings to ensure the correct code style is used with any new features and changes.
 7. One by one, remove the custom rules and refactor the codebase as required. You could create stories for each of them and plan in one or two every sprint or whenever your team has time to work on clearing technical debt.
 
+## Publishing to npmjs.com
+
+### prerequisites
+
+You’ll need an account on npmjs.com and that has access to this package.
+Account needs 2FA enabled for authorization & publishing.
+
+### Step 1 - Bump version
+
+Create a new branch in which you bump the version, in here you'll also need to update [CHANGELOG.md](https://github.com/Mobiliteitsfabriek/eslint-config-mig-react-native/blob/main/CHANGELOG.md). Move everything from [Unreleased] to the new version that you're creating. Using `npm version` you can bump the version.
```

Comment:
it would be great to have the naming pattern for branches, for example, `version-bump/*`

Reply:
Good idea! I will add it to the documentation